pkgs/os-specific/linux/util-linux/default.nix

```diff
@@ -57,7 +60,8 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkg-config ];
   buildInputs =
     [ zlib pam libcap_ng ]
-    ++ lib.filter (p: p != null) [ ncurses systemd ];
+    ++ lib.filter (p: p != null) [ ncurses systemd ]
+    ++ lib.optional stdenv.hostPlatform.isStatic [ audit ];
```

Comment:
This should be optionals or optional with a single value. Personally, I prefer to never use optional since it allows errors like this.

Reply:
Fixed, I'll try to avoid it too in the future, thanks !